sigle/src/ui/IconButton.ts

```diff
@@ -6,8 +6,16 @@ export const IconButton = styled('button', {
   alignItems: 'center',
   fontSize: '$2',
 
+  '& svg': {
```

Comment:
Can you please explain what was not working before?

Reply:
I think here I was testing to confirm the color would change, as it seemed not to work with `css` prop, but I think this was actually because the page hadn't refreshed - will remove this!